extensions/sql-database-projects/src/test/netCoreTool.test.ts

```diff
@@ -48,13 +47,19 @@ describe('NetCoreTool: Net core tests', function (): void {
 
 		if (os.platform() === 'win32') {
 			// check that path should start with c:\program files
-			let result = isNullOrUndefined(netcoreTool.netcoreInstallLocation) || netcoreTool.netcoreInstallLocation.toLowerCase().startsWith('c:\\program files');
+			let result = netcoreTool.netcoreInstallLocation === undefined || netcoreTool.netcoreInstallLocation === null || netcoreTool.netcoreInstallLocation.toLowerCase().startsWith('c:\\program files');
```

Comment:
These checks can be simplified to `!netcoreTool.netcoreInstallLocation` since that will handle both null and undefined. Also good to simplify this since we try not to use null in this codebase

Reply:
and same for below